src/Console/Command/PackageReleaseCommand.php

```diff
@@ -0,0 +1,380 @@
+                    OutputFormatter::escape($release_branch)
+                )
+            );
+        }
+
+        $message = $input->getOption('message');
+        if ($message === '') {
+            $message = sprintf(
+                'Release version %s.',
+                $next_version
+            );
+        }
+        $this->startCommand($output);
+        $success = $this->manager->createReleaseTag(
+            $next_version,
+            $message
+        );
+        if ($success) {
+            $this->handleSuccess(
+                $output,
+                sprintf(
+                    'tagged release with message <variable>%s</variable>',
+                    OutputFormatter::escape($message)
+                )
+            );
+        } else {
+            $this->handleError(
+                $output,
+                sprintf(
+                    'failed to create release tag for <variable>%s</variable>',
+                    OutputFormatter::escape($next_version)
+                ),
+                $this->manager->getLastError()
+            );
+            return 1;
+        }
+
+        $this->startCommand($output);
+        if ($this->manager->pushTagToRemote($next_version, $remote)) {
+            $this->handleSuccess(
+                $output,
+                sprintf(
+                    'pushed tag to <variable>%s</variable>',
+                    OutputFormatter::escape($remote)
+                )
+            );
+        } else {
+            $this->handleError(
+                $output,
+                sprintf(
+                    'could not push tag <variable>%s</variable> to remote '
+                    . '<variable>%s</variable>',
+                    OutputFormatter::escape($next_version),
+                    OutputFormatter::escape($remote)
+                ),
+                $this->manager->getLastError()
+            );
+            return 1;
+        }
+
+        $this->startCommand($output);
+        if ($this->manager->deleteBranch($release_branch)) {
+            $this->handleSuccess(
+                $output,
+                sprintf(
+                    'removed release branch <variable>%s</variable>',
+                    OutputFormatter::escape($release_branch)
+                )
+            );
+        } else {
+            $this->handleError(
+                $output,
+                sprintf(
+                    'could not delete release branch <variable>%s</variable>',
+                    OutputFormatter::escape($release_branch)
+                ),
+                $this->manager->getLastError()
+            );
+            return 1;
+        }
+
+        $output->writeln([
+            '',
+            '<info>Success!</info>',
+            '',
+            'The composer repository will automatically update. It may take a '
+            . 'few minutes for the release to appear at '
+            . '<link>https://composer/</link>.',
+            ''
+        ]);
+    }
```

Comment:
There's an implicit return at the end of this function that throws an error because it's returning null instead of an integer:

```
Fatal error: Uncaught TypeError: Return value of Silverorange\PackageRelease\Console\Command\PackageReleaseCommand::execute() must be of the type integer, none returned in /so/packages/tools/work-meg/src/Console/Command/PackageReleaseCommand.php on line 322

TypeError: Return value of Silverorange\PackageRelease\Console\Command\PackageReleaseCommand::execute() must be of the type integer, none returned in /so/packages/tools/work-meg/src/Console/Command/PackageReleaseCommand.php on line 322

Call Stack:
    0.0004     358352   1. {main}() /so/packages/tools/work-meg/bin/package-release:0
    0.0102    1197504   2. Symfony\Component\Console\Application->run() /so/packages/tools/work-meg/bin/package-release:56
    0.0221    1440544   3. Symfony\Component\Console\Application->doRun() /so/packages/tools/work-meg/vendor/symfony/console/Application.php:143
    0.0222    1440544   4. Symfony\Component\Console\Application->doRunCommand() /so/packages/tools/work-meg/vendor/symfony/console/Application.php:241
    0.0223    1440544   5. Silverorange\PackageRelease\Console\Command\PackageReleaseCommand->run() /so/packages/tools/work-meg/vendor/symfony/console/Application.php:865
    0.0228    1445216   6. Silverorange\PackageRelease\Console\Command\PackageReleaseCommand->execute() /so/packages/tools/work-meg/vendor/symfony/console/Command/Command.php:252

```

Reply:
Fixed in https://github.com/silverorange/package_release/pull/12/commits/1eab3cfa60256ba0cec041dcf41911276ef075cc